cinema/views.py

```diff
@@ -56,3 +106,27 @@
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderPagination(PageNumberPagination):
+    page_size = 5
+    max_page_size = 100
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.prefetch_related(
+        "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
+    )
+    serializer_class = OrderSerializer
+    pagination_class = OrderPagination
+
+    def get_queryset(self):
+        return Order.objects.filter(user=self.request.user)
```

Comment:
The `get_queryset` method in `OrderViewSet` filters orders by the current user. Ensure that the viewset requires authentication to prevent unauthorized access to this endpoint.